Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,46 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            while (InteractiosnManager.ContinueRunningProgram)
+            {
+                InteractiosnManager.PerformInteractions();
+            }
+        }
+    }
+
+    public static class InteractiosnManager
+    {
+        public static bool ContinueRunningProgram = true;
+        public static void PerformInteractions()
+        {
+            var name = Prompter("What is your full name?");
+            var age = int.Parse(Prompter("What is your age?"));
+            var weight = double.Parse(Prompter("What is your weight in kg?"));
+            var height = double.Parse(Prompter("What is your height in cm?"));
+            var bmi = CalculateBmi(height, weight);
+
+            Console.WriteLine($"{name} is {age} years old, his weight is {weight} kg and his height is {height} cm.");
+            Console.WriteLine($"{name} has a BMI of {bmi}");
+
+            var response = Prompter("Would you like to run the program again? (y/n)");
+            ContinueRunningProgram = response.ToLower() == "y" ? true : false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896164)